website/addons/wiki/templates/wiki-bar-modal-help.mako

```diff
@@ -0,0 +1,18 @@
+<div class="modal fade" id="wiki-help-modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true"></span>&times;</button>
+                    <h3 class="modal-title">Wiki syntax help</h3>
+                </div>
+                <div class="modal-body">
+                  <p>
+                    Wiki uses the Markdown syntax. This gives you many options but can be very simple as well. To see more information and examples go to markdown <a href="https://daringfireball.net/projects/markdown/">guides.</a>
```

Comment:
Okay, thank you. This page didn't exist when I first took the ticket and I had to wait for it to be made.